### PR TITLE
[jit] Fix stdout capturing, remove some expect files

### PR DESCRIPTION
**Pull request by driazati**
The expect files were making this fail in `pytest` for some reason, this removes them since they're not needed

Differential Revision: [D15855454](https://our.internmc.facebook.com/intern/diff/15855454/)

### CODE REVIEW

**Review by jamesr66a — APPROVED** [review]
Not great, but not terrible

**Comment by SsnL** [comment]
This broke windows CI.

**Comment by SsnL** [comment]
@yf225 in case you want to unland this too.

**Comment by ezyang** [comment]
I agree with this assessment.

File: test/jit_utils.py [hunk 1]
```diff
@@ -258,50 +258,52 @@ def run_pass(self, name, trace):
...
         if capture_output:
-            with self.capture_stdout() as captured:
-                outputs_ge = ge(*inputs)
             if not IS_WINDOWS:
-                self.assertExpected(captured[0], subname='stdout')
+                with self.capture_stdout() as script_stdout:
+                    outputs_ge = scripted_fn(*inputs)
+                with self.capture_stdout() as python_stdout:
+                    outputs_ge = scripted_fn(*inputs)
+                self.assertEqual(script_stdout, python_stdout)
```
Comment (SsnL) [inline test/jit_utils.py]: in particular, this is missing an `else`

**Pull request closed without merging**